Client/Application/AppLaunchUtil.swift

```diff
@@ -131,6 +131,7 @@ class AppLaunchUtil {
         // increase session count value
         profile.prefs.setInt(sessionCount + 1, forKey: PrefsKeys.Session.Count)
         UserDefaults.standard.set(Date.now(), forKey: PrefsKeys.Session.Last)
+        UserConversionMetrics.didStartNewSession()
```

Comment:
I thought at first to do it that way, like before `UserDefaults.standard.set(Date.now(), forKey: PrefsKeys.Session.Last)` is called, to check if the current date is a different day in the first week of usage than the saved date and update the counter for a PrefsKeys.Session.distinctDaysSessions key, and then add 2 new keys for the searches and do something similar, but the activation criteria for this event are kinda specific so if BA will want this event to happen in different conditions for the first week of usage it would be easy to change and would not require other flags to be added